src/js/libcs/Essentials.js

```diff
@@ -183,13 +183,13 @@ niceprint.errorTypes = ["_?_", "_??_", "_???_", "___"];
 //Distinct form evaluator for code clearness :-)
 //*******************************************************
 function evalmyfunctions(name, args, modifs) {
-    var tt = myfunctions[name];
+    const tt = myfunctions[name];
     if (tt === undefined) {
         return nada;
     }
 
-    var set = [],
-        i;
+    const set = [];
+    let i;
 
     for (i = 0; i < tt.arglist.length; i++) {
```

Comment:
Change `for (i = 0;` to `for (let i = 0;` here and in the following two `for (i = 0;` occurrences below and remove the `let i;` above.